normandy/classifier/tests/test_api.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from normandy.base.utils import Whatever
+from normandy.recipes.tests import RecipeActionFactory, RecipeFactory
+
+@pytest.mark.django_db
+class TestFetchBundleAPI(object):
+
+    def test_it_works(self, client):
```

Comment:
Man, I ___hate___ dependency injection. This is fine because it's how they do it, but jeez.